### PR TITLE
Lebanon: Updated regions and new locality polygons

### LINKED ISSUE #629: Lebanon: Update region records once new administrative divisions are implemented
Currently, Who's On First has six polygon features and one point feature (to be deprecated) at the `region` level for Lebanon. The Lebanese government approved two new regions in Northern Lebanon; **Akkar** and **Baalbeck-Herme**. These regions will be implemented in the near future, but for now, the government only recognizes six administrative regions. 

When these regions are implemented, add a `edtf:cessation` date and `mz:is_current` values to the records of [Beqaa](https://whosonfirst.mapzen.com/spelunker/id/85673479/) and [North Lebanon](https://whosonfirst.mapzen.com/spelunker/id/85673483/) with superseding pointers to two new records each.

References:
- https://en.wikipedia.org/wiki/Governorates_of_Lebanon
- https://now.mmedia.me/lb/en/commentary/the_name_game

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #629 

This PR includes:

- New locality polygons for Beirut and Tripoli
- Moves any locality points that fall within the new locality polygons to new neighbourhood records
- Updated region records, splitting North Lebanon and Beqaa into Beqaa, Baalbek-Hermel, North Lebanon, Akkar
  - Since geometries substantially changed and the original Beqaa and North Lebanon are new divisions, includes superseding work and new features (rather than adjusting in place and adding two new records)

To do:
- [x] Add name translations to Beqaa and North Lebanon
- [x] Flag Beirut locality and region as coterminous
- [x] Take care of duplicate Tripoli records (1125791429 / 421201875)

### CODE REVIEW

**Comment by stepps00** [comment]
Will validate geometries and GeoJSON once approved, then merge.

File: data/112/579/142/9/1125791429.geojson [hunk 1]
```diff
@@ -307,22 +334,24 @@
...
...532021],[35.86161311858589,34.41085782771219],[35.86608764248003,34.40963075861478],[35.87098678903908,34.40025640949227],[35.87024208469062,34.39132161476137],[35.85758498583981,34.39279553859921],[35.85455085625693,34.39057664356388],[35.84675140029579,34.39108081551292],[35.84046921413271,34.39330422765678],[35.84000000000023,34.40013089208906],[35.83902405999878,34.40820432013025],[35.8379526200636,34.41163269954491],[35.8351403474902,34.41658115161647],[35.82820717207909,34.41952857190548],[35.82471063804469,34.42092124797171],[35.82296030352307,34.42509063848379],[35.82112787611586,34.43054628014886],[35.8187316759272,34.4386824114119],[35.81947231957058,34.44149502758234],[35.82268687754681,34.4452690966138],[35.82396451595253,34.44892061014099],[35.82505416900005,34.45113754300007],[35.82713699300007,34.45322036700003],[35.82463836700003,34.45738983200005],[35.82502746600005,34.45863723800005],[35.82669448900003,34.45866775500008],[35.83005523700007,34.45366668700007],[35.83336257900004,34.45363998400006],[35.84172058100006,34.45780563400007],[35.84502792400008,34.45777893100006],[35.84591674800004,34.45694351200007],[35.85407829210974,34.45697085423453]]],"type":"Polygon"}
```
Comment (nvkelso) [inline data/112/579/142/9/1125791429.geojson]: Neat! I learned that El Mina is a different locality. Carry on :)

**Comment by stepps00** [comment]
All records and geometries validate, merging.

**Comment by stepps00** [comment]
All hierarchy and PIP work completed in b8be9a9. I'll merge once QA'd.

cc @nvkelso

**Pull request merged**